### DISCUSSION TITLE
how to disable animation from 0 to current value by Y for Jetpack Compose

### DISCUSSION THREAD

**Discussion by yatoshk** [patrykandpatrick/vico#906] [Questions] [answered]
### Question

I tried to understand from the existing answers but they did not help

          var refreshDataset = remember { mutableIntStateOf(0) }
          Box(modifier = Modifier
          .padding(top = 10.dp, end = 20.dp)
          .fillMaxSize()) {
          val modelProducer = remember { CartesianChartModelProducer.build() }
          val marker = DefaultCartesianMarker(label = TextComponent.build())
          LaunchedEffect(key1 = refreshDataset.intValue) {
            modelProducer.tryRunTransaction {
                lineSeries {
                    series( y = ManagerGraph.handlerTargetValues)
                    series( y = ManagerGraph.handlerValues )
                    series( y = listOf(ManagerGraph.settings.heightGraph)  )
                }
            }

            if (isGetValues.value && ManagerGraph.settings.targetValue > ManagerGraph.handlerValues.last())
            {
                ManagerGraph.imitationGetValues()
            } else if (ManagerGraph.settings.targetValue <= ManagerGraph.handlerValues.last()){
                isGetValues.value = !isGetValues.value
                resultSeconds.value = "${ManagerGraph.handlerTargetValues.size - 1} sec"
            }

            delay(1000L)
            if (refreshDataset.intValue < 10000)
                refreshDataset.intValue++
            else
                refreshDataset.intValue = 0
        }

        CartesianChartHost(
            rememberCartesianChart(
                rememberLineCartesianLayer(lines = chartColors.map { color: Color ->
                    rememberLineSpec(
                        shader = DynamicShader.color(
                            color
                        ), backgroundShader = null
                    )
                }),
                startAxis = rememberStartAxis(
                    titleComponent =
                    rememberTextComponent(
                        background = rememberShapeComponent(Shape.Pill, Color.Transparent),
                        color = Color.White,
                        padding = Dimensions.of(horizontal = 8.dp, vertical = 2.dp),
                        margins = Dimensions.of(top = 4.dp),
                        typeface = Typeface.MONOSPACE,
                    ),
                    title = ""
                ),
                bottomAxis = rememberBottomAxis(
                    tickLength = 15.dp,
                    titleComponent =
                    rememberTextComponent(
                        background = rememberShapeComponent(Shape.Pill, Color.Transparent),
                        color = Color.White,
                        padding = Dimensions.of(horizontal = 8.dp, vertical = 2.dp),
                        margins = Dimensions.of(top = 4.dp),
                        typeface = Typeface.MONOSPACE,
                    ),
                    title = "seconds"
                ),
            ),
            modifier = Modifier
                .padding(top = 10.dp, bottom = 240.dp)
                .fillMaxSize(),
            marker = marker,
            modelProducer = modelProducer,
            zoomState = rememberVicoZoomState(
                zoomEnabled = false,
                initialZoom = Zoom.Content
            ),
            runInitialAnimation = false,

        )
    }

https://github.com/user-attachments/assets/9d764cb2-7c3e-474c-abc4-74dd3eca1048

### Vico version(s)

2.0.0-alpha.21

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment, marked as answer]
Hello, @Yatoshk. You can disable this animation like so:
```kt
CartesianChartHost(
  ...
  diffAnimationSpec = null,
)
```

**Reply by yatoshk** [reply]
thanks a lot!
